nautobot_device_lifecycle_mgmt/views.py

```diff
@@ -31,6 +38,34 @@
 GREEN, RED, GREY = (f"#{ColorChoices.COLOR_LIGHT_GREEN}", f"#{ColorChoices.COLOR_RED}", f"#{ColorChoices.COLOR_GREY}")
 
 
+#
+# Custom ObjectFieldsPanel for HardwareLCM
+#
+class HardwareLCMFieldsPanel(object_detail.ObjectFieldsPanel):
```

Comment:
Is this required? I replaced this with a base `ObjectFieldsPanel` locally and saw the same behavior for devices.